internal/core/software_renderer.rs

```diff
@@ -299,6 +301,22 @@ impl<const MAX_BUFFER_AGE: usize> Renderer for SoftwareRenderer<MAX_BUFFER_AGE>
         fonts::register_bitmap_font(font_data);
     }
 
+    #[cfg(feature = "systemfonts")] // FIXME: just because of the Error
```

Comment:
Yeah, this is a silly copy & paste error :). It comes from the trait where this function is cfg-guarded only because of the error type in std. But that doesn't apply here.